src/cli/format.js

```diff
@@ -426,11 +426,21 @@ async function formatFiles(context) {
     if (numberOfUnformattedFilesFound === 0) {
       context.logger.log("All matched files use Prettier code style!");
     } else {
-      context.logger.warn(
-        context.argv.write
-          ? "Code style issues fixed in the above file(s)."
-          : "Code style issues found in the above file(s). Forgot to run Prettier?"
-      );
+      if (numberOfUnformattedFilesFound === 1) {
+        context.logger.warn(
+          context.argv.write
+            ? "Code style issues fixed in the above file."
+            : "Code style issues found in the above file. Forgot to run Prettier?"
+        );
+      } else {
+        context.logger.warn(
+          context.argv.write
+            ? "Code style issues fixed in the above file(s)."
+            : "Code style issues found in " +
+                numberOfUnformattedFilesFound +
+                " files. Forgot to run Prettier?"
+        );
+      }
```

Comment:
Strictly speaking, this is a breaking change in CI. We may break downstream snapshot with this change and the value is quite small compared to the risks. +1 on splitting this PR into two. I would not include this particular change in Prettier 2.x.

Reply:
Okay I shall revert to not address this change.

Reply:
Feel free to create a new PR with this – we can discuss it separately 🙂

Reply:
Shall I create a new PR for that change, once the website changes are addressed?

Reply:
There’s no need to wait for this PR to be merged. So feel free to create a new PR whenever it is convenient for you 🙂

Reply:
I have created another branch for the renaming Zeit change. #12441